envoy.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM envoyproxy/envoy:latest
+
+COPY envoy.yaml /etc/envoy.yaml
+COPY envoy-entrypoint.sh /
+
+RUN chmod 500 /envoy-entrypoint.sh
+
+RUN apt-get update && \
+    apt-get install gettext -y
+
+ENTRYPOINT ["/envoy-entrypoint.sh"]
```

Comment:
I think we can dispense with the entrypoint script, and just run /usr/local/bin/envoy -c /etc/envoy.yaml here

Reply:
Sounds good. Will change this in the follow-up